code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/lb/LoadBalancerConfigSetListenersActionHandler.java

```diff
@@ -46,7 +54,7 @@ public HandlerResult handle(ProcessState state, ProcessInstance process) {
             // create a new set
             createNewListenerMaps(config, newListenerIds);
         }
-        return null;
+        return objectManager.reload(config);
```

Comment:
Actually, I was going to comment that I don't think it's necessary to reload.  But figured it doesn't do too much harm (at least until we need to worry about performance testing)